common/src/main/java/com/wynntils/mc/utils/objects/ServerIcon.java

```diff
@@ -101,24 +101,25 @@ private synchronized void loadServerIcon(ResourceLocation destination) {
             return;
         }
 
-        NativeImage nativeImage;
         try {
-            nativeImage = NativeImage.fromBase64(iconString);
+            try (NativeImage nativeImage = NativeImage.fromBase64(iconString)) {
```

Comment:
Not sure about this but it should be fine because it is disabled.

Reply:
I looked at the NativeImage code and it really looked like it should have it's close() method called. So I agreed with IntelliJ and said that this was an autocloseable miss.